fetch-key-vault-secrets/action.yml

```
@@ -0,0 +1,31 @@
+name: Fetch Key Vault Secrets
+
+description: Action to Fetch Secrets stored in Azure Key Vault
+
+inputs:
+  keyvault:
+    description: Name of the Key Vault the secrets are being stored
+    required: true
+
+  secrets:
+    description: Comma-separated list of secret names to fetch values from
+    required: true
+
+runs:
+  using: "composite"
```

```suggestion
  using: composite
```

YAML strings don't need quotes.